src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -178,6 +179,24 @@ public class SetAzureSqlManagedInstance : ManagedInstanceCmdletBase
         [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
         public SwitchParameter Force { get; set; }
 
+        /// <summary>
+        /// Gets or sets the instance compute generation
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The compute generation for the instance.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter(Constants.ComputeGenerationGen5)]
+        public string ComputeGeneration { get; set; }
+
+        /// <summary>
+        /// Gets or sets the instance SKU name
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The SKU name for the instance e.g. 'GP_Gen5','BC_Gen5'.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter(Constants.GeneralPurposeGen5, Constants.BusinessCriticalGen5)]
+        public string SkuName { get; set; }
```

Comment:
I think this should be a different parameter set than compute&family params.

Reply:
As we discussed offline, we will remove SkuName from parameters.